app/src/main/java/com/graddex/graddex/PokemonDetailsFragment.kt

```diff
@@ -0,0 +1,96 @@
+package com.graddex.graddex
+
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.os.bundleOf
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.ViewModelProvider
+import coil.load
+import com.graddex.graddex.R.string.*
+import com.graddex.graddex.models.PokemonDetailsViewModel
+import com.graddex.graddex.databinding.FragmentPokemonDetailsBinding
+import java.util.*
+
+private const val pokemonKey: String = "pokemonKey"
+
+class PokemonDetailsFragment(args: Bundle) : Fragment() {
+
+    constructor(id: String) : this(bundleOf(pokemonKey to id))
+
+    private val pokemonId: String = args.getString(pokemonKey, "")
+
+    private lateinit var binding: FragmentPokemonDetailsBinding
+    private lateinit var viewModel: PokemonDetailsViewModel
+
+    override fun onCreateView(
+            inflater: LayoutInflater, container: ViewGroup?,
+            savedInstanceState: Bundle?
+    ): View? {
+
+        Log.d(tag, "PokemonID: $pokemonId")
+
+        // Inflate the Pokemon Details fragment layout
+        binding = DataBindingUtil.inflate<FragmentPokemonDetailsBinding>(
+            inflater, R.layout.fragment_pokemon_details, container, false
+        )
+
+        // Call the Pokemon Details view model for specified Pokemon ID
+        viewModel = ViewModelProvider(this)
+            .get(PokemonDetailsViewModel::class.java)
+
+        binding.statusText.text = getString(loading)
+        viewModel.syncPokemonDetails(pokemonId)
```

Comment:
Could you move this to `onActivityCreated` too please

Reply:
moved :)